src/Action/DashboardAction.php

```diff
@@ -39,20 +39,33 @@
      */
     private $pool;
 
+    /**
+     * @var string
+     */
+    private $redirect;
+
     public function __construct(
         array $dashboardBlocks,
         BreadcrumbsBuilderInterface $breadcrumbsBuilder,
         TemplateRegistryInterface $templateRegistry,
-        Pool $pool
+        Pool $pool,
+        string $redirect = ''
```

Comment:
We still support PHP 5.6 on the stable branch :/

Reply:
What do we need to change to support PHP 5.6 ?

Reply:
This:

```suggestion
        $redirect = ''
```